packages/2d/src/components/Latex.ts

```diff
@@ -29,6 +29,35 @@ export interface LatexProps extends ImgProps {
   renderProps?: SignalValue<OptionList>;
 }
 
+/**
+ * A node for rendering equations with LaTeX.
+ *
+ * @preview
+ * ```tsx editor
+ * import {Latex} from '@motion-canvas/2d/lib/components';
+ * import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+ * import {waitFor} from '@motion-canvas/core/lib/flow';
+ * import {createRef} from '@motion-canvas/core/lib/utils';
+ *
+ * export default makeScene2D(function* (view) {
+ *   const tex = createRef<Latex>();
+ *   view.add(
+ *     <Latex
+ *       ref={tex}
+ *       // Note how this uses \color to set the color.
+ *       tex="{\color{white} ax^2+bx+c=0 \implies x=\frac{-b \pm \sqrt{b^2-4ac}}{2a}}"
+ *       y={0}
+ *       width={600} // height and width can calculate based on each other
+ *     />,
+ *   );
+ *
+ *   yield* waitFor(2);
+ *   yield* tex().opacity(0, 1);
+ *   yield* waitFor(2);
+ *   yield* tex().opacity(1, 1);
```

Comment:
I know this has been copied from the latex page but I don't think the animation is necessary in this case, especially since `Latex` doesn't have any special properties to animate.
```suggestion
```